src/FormMigration/Steps/FeeRecovery.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+class FeeRecovery extends FormMigrationStep
+{
+
+    /**
+     * @unreleased
+     */
+    public function process()
+    {
+        $feeRecoverySettings = $this->formV2->getFeeRecoverySettings();
+
+        if (empty($feeRecoverySettings) || (
+                $feeRecoverySettings['useGlobalSettings'] === true &&
+                !give_is_setting_enabled(give_get_option('give_fee_recovery', 'disabled'))
+            )) {
+            return;
+        }
+
+        $feeRecoveryBlock = BlockModel::make([
+            'name' => 'givewp-fee-recovery/fee-recovery',
+            'attributes' => $feeRecoverySettings,
```

Comment:
Will the block work properly if only the useGlobalSettings is set to true?  (without setting the other attributes). Lets just make sure of that.

Reply:
No. The block is broken now. Populating the array with the global options now: 7d8baf91a96296bde35c1c6fc2b1c7674e886ee4